odoo/modules/loading.py

```diff
@@ -246,6 +246,12 @@ def load_module_graph(cr, graph, status=None, perform_checks=True,
             # update made to the schema or data so the tests can run
             # (separately in their own transaction)
             cr.commit()
+            if model_names:
+                cr.execute("""select model from ir_model where id NOT IN (select distinct model_id from ir_model_access) AND model IN %s""", (tuple(model_names),))
```

Comment:
newlines are cheap, and readability counts
```suggestion
                query = """ SELECT model FROM ir_model WHERE model IN %s
                            AND id NOT IN (SELECT DISTINCT model_id FROM ir_model_access) """
                cr.execute(query, [tuple(model_names)])
```